pkg_resources/__init__.py

```diff
@@ -2656,15 +2666,23 @@ def _warn_legacy_version(self):
 
         warnings.warn(tmpl.format(**vars(self)), PEP440Warning)
 
+    def _make_missing_version_message(self):
```

Comment:
This only seems to be used in one place, I think we can inline it for now (unless I'm missing something).